blueprints/hr_assistant/greeting.py

```diff
@@ -12,7 +12,8 @@ def greet(request, responder):
                     "an employee's individual information (eg. Is Mia married?), "
                     "some employee statistic (eg. average salary of females) or "
```

Comment:
replace `or` with comma here. maybe we should shorten this greeting?

Reply:
"You can ask me about an individual employee, company's employee demographics or general policy questions"